mmseg/datasets/coco_stuff.py

```diff
@@ -0,0 +1,223 @@
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class COCOStuffDataset(CustomDataset):
+    """COCO-Stuff dataset.
+
+    In segmentation map annotation for COCO-Stuff, 0 stands for unlabelled,
+    which is not included in 182 categories. ``reduce_zero_label`` is fixed to
+    True. The ``img_suffix`` is fixed to '.jpg' and ``seg_map_suffix`` is fixed
+    to '.png'.
+    """
```

Comment:
Could you please describe the difference between 182 class label ID and 171 train label ID?
Why we use 171 classes in the training and evaluation for COCO Stuff in the sematic segmentation task.